src/components/MultiLink/MultiLink.vue

```diff
@@ -157,8 +157,8 @@ export default Vue.extend({
 
     svgDimensions() {
       return {
-        height: document.body.clientHeight,
-        width: document.body.clientWidth - 256,
+        height: this.$vuetify.breakpoint.height,
+        width: this.$vuetify.breakpoint.width - 256,
```

Comment:
This looks pretty cool. I read the docs [here](https://vuetifyjs.com/en/features/breakpoints/#breakpoint-service) but I don't fully understand what `this.$vuetify` is meant for. I'm going to ask Jared for a formal review in part so we all can learn something new here.

Reply:
This use case, checking the size of the window (it will also give you which breakpoint we're in sm, md, lg, etc.) is for designing custom reactive elements that scale with the window. It's like how you can use a css framework for those breaks, but it gives you access to the info in javacript.

Reply:
I think this is probably the best way to do this because of how vuetify handles responsive as well as css in general. What I mean is CSS breakpoints would probably not make sense.